NDTensors/src/lib/GradedAxes/src/unitrangedual.jl

```diff
@@ -62,3 +64,8 @@ BlockArrays.blockaxes(a::UnitRangeDual) = blockaxes(nondual(a))
 BlockArrays.blockfirsts(a::UnitRangeDual) = label_dual.(blockfirsts(nondual(a)))
 BlockArrays.blocklasts(a::UnitRangeDual) = label_dual.(blocklasts(nondual(a)))
 BlockArrays.findblock(a::UnitRangeDual, index::Integer) = findblock(nondual(a), index)
+
+blocklabels(a::UnitRangeDual) = blocklabels(nondual(a))
```

Comment:
Is this right? Shouldn't it be `blocklabels(a::UnitRangeDual) = dual.(blocklabels(nondual(a)))`?

Reply:
It is a convention to be set. I hesitated when I wrote it, maybe taking the dual is easier to predict. I will check where I call this method.

Reply:
Yeah I think that definition is a bit confusing. We could define `nondual_blocklabels` that has this behavior but I'm not sure that is needed since you can just call `blocklabels(nondual(a))` explicitly.

Reply:
I agree that seeing either `blocklabels(nondual(a))` or a function name like `blocklabels_ignore_dual` would be less confusing if I came across it in the code.

Reply:
So indeed I had a good reason, changing it messes up the fusion trees -_-
I plan to commit the change, but it requires a bit of work.

Reply:
No worries, just as long as we are agreed on a design plan. I definitely understand this PR requires coordinating with your FusionTensor work.